wolfcrypt/src/misc.c

```diff
@@ -201,7 +201,13 @@ STATIC INLINE void ForceZero(const void* mem, word32 len)
     volatile byte* z = (volatile byte*)mem;
 #if defined(WOLFSSL_X86_64_BUILD) && defined(WORD64_AVAILABLE)
     volatile word64* w;
+    #ifndef WOLFSSL_UNALIGNED_64BIT_ACCESS
+        word32 l = (8 - ((size_t)z & 0x7)) & 0x7;
```

Comment:
Perhaps use `sizeof(word64)` and `sizeof(word64)-1` here instead of 8 and 7.

Reply:
I'll make the change.
Todd asked me to fix this issue and I appreciate you changing your pull request!